packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -66,7 +66,11 @@ class CryptoWatchPriceFeed extends PriceFeedInterface {
   }
 
   getCurrentPrice() {
-    return this.invertPrice ? this._invertPriceSafely(this.currentPrice) : this.currentPrice;
+    return this.twapLength
+      ? this.currentPrice
+      : this.invertPrice
+      ? this._invertPriceSafely(this.currentPrice)
```

Comment:
I think this is more readable if you put parantheses around the `(this.invertPrice ? this._invertPriceSafely(this.currentPrice) : this.currentPrice)` part!

Reply:
Hm I had this, but linting seems to disagree.